### ISSUE TITLE
Day One: Slider + emojis: retaining it

### ISSUE THREAD

**Issue by paulinahaduong** [GSE-CCL/getting-unstuck-web#161]
Lower priority: when KB and I submit our times + emojis, then click refresh, the data seems to be lost (but slider/emojis still appear). When I view [the page ](https://gettingunstuck.gse.harvard.edu/project/409448427/view) in incognito, the slider/emojis aren't visible - they flash and then vanish. Karen replicated this issue; I also replicated it in Firefox.

- [x] Is it still recording the data?

- [ ] How can we get it to retain?

**Comment by jsarchibald** [comment]
Yes, still recording data! I made a change in the JS last night that apparently had side effects. So that will be in prodpatch13

**Comment by jsarchibald** [comment]
```js
Uncaught ReferenceError: can't access lexical declaration 'saved' before initialization
    init_reflection https://gettingunstuck.gse.harvard.edu/static/js/project_results.js:30
    loaded_pr https://gettingunstuck.gse.harvard.edu/static/js/project_results.js:87
```

**Comment by jsarchibald** [comment]
Should be fixed by #167
